config/routes.rb

```diff
@@ -221,6 +221,10 @@
       post "batch/:programme_type/:vaccine_method", action: :update_batch
     end
 
+    resource :manage_consent_reminders,
```

Comment:
This will generate a path with underscores, however our convention is to use dashes: `path: "manage-consent-reminders"`.